### ISSUE TITLE
Feature：Add click event on the root UL element

### ISSUE THREAD

**Issue by synapse** [minop1205/react-dnd-treeview#64]
Hey @minop1205 , just want to say that by far this is the best and most versatile tree component so far.
I would like to ask you if there's a way to add a click event on the root UL in order to unselect everything?
You can see that same functionality in VSCode by selecting a folder/file and then hitting the empty space below.

**Comment by minop1205** [comment]
@synapse Thanks for the feature request!

At the moment, there is no way to set onClick for the root UL.

In this case (deselecting the root UL by clicking on it), it is possible to wrap the entire Treeview and set the event handler there, as shown below.

https://codesandbox.io/s/multiple-selections-issue-64-epqtf?file=/src/App.tsx

However, since this is a hack-like approach, I think it would be more useful to be able to add arbitrary properties to the root UL as you say.

For example, here is an code

```
<Tree
  {... .props}
  rootProps={{.
    onClick={() => console.log("foo")}
    // and other props
  }}
/>
```

I think a bit about how to implement this.

**Comment by minop1205** [comment]
Merged: https://github.com/minop1205/react-dnd-treeview/pull/72

**Comment by minop1205** [comment]
I added an API called rootProps to set arbitrary properties on the root node, and published it as v1.5.9.

You can use rootProps to set the onClick callback as follows

```
<Tree
  {... .props}
  rootProps={{.
    onClick: (e) => {
      if (e.target === e.currentTarget) {
        // do something
      }
    },
  }}
/>
```

The following is a sample of clicking on the root area to deselect it.
https://codesandbox.io/s/multiple-selections-ts-eud8c?file=/src/App.tsx
